### ISSUE TITLE
Saving points doesn't work

### ISSUE THREAD

**Issue by M0pf** [rengolin/cardgamescores#14]
Hi, on my S4 mini with Android 4.2.2 the saving function doesn't work. If I press "last game" I get the old settings (with names and so on), but the scores are on zero again. 

But a great up! Would like to have this fix :-)

**Comment by rengolin** [comment]
Hi, "previous" game wasn't meant to save the scores, just meant to save the players. For instance, if you always play with the same people (as a family), then you save entering the names again.

**Comment by M0pf** [comment]
Hm. Okay. But this function would be really nice, if you play a game for more than one day. Could you add the saving score feature? For example a button "Reload last game and score"? The this app would be perfect!

**Comment by rengolin** [comment]
Indeed, your not the first one to ask for that. Though I have to say I am not very good at android development. :-)

Let's keep this task open as a request, and I'll update when I can. Feel free to make a pull request if you're versed in apps. 

Thanks!

**Comment by StefanBrand** [comment]
I didn't know that the score is not saved so, when I tried to convince my friends about the app, it embarrassed me...
